mypy-baseline.txt

```diff
@@ -847,7 +847,7 @@ src/inmanta/ast/statements/define.py:0: error: Return type "None" of "register_t
 src/inmanta/ast/statements/define.py:0: error: Incompatible types in assignment (expression has type "Namespace | None", variable has type "Namespace")  [assignment]
 src/inmanta/agent/handler.py:0: error: Argument 3 to "log" of "LogLine" has incompatible type "**dict[str, object]"; expected "datetime | None"  [arg-type]
 src/inmanta/agent/handler.py:0: error: Returning Any from function declared to return "bool"  [no-any-return]
-src/inmanta/agent/handler.py:0: error: Argument 2 to "chain_future" has incompatible type "concurrent.futures._base.Future[T]"; expected "asyncio.futures.Future[Any]"  [arg-type]
+src/inmanta/agent/handler.py:0: error: Argument 2 to "chain_future" has incompatible type "concurrent.futures._base.Future[T]"; expected "_asyncio.Future[Any]"  [arg-type]
```

Comment:
This is a bug in tornado. It will be fixed in their next release: https://github.com/tornadoweb/tornado/commit/385af837f278ec0a6489cb61db158960e648fa94